### PR TITLE
Add patch for pickling acm certificate

### PR DESCRIPTION
**Pull request by giograno**
This PR adds a patch to pickle the `CertBundle` models.

### CODE REVIEW

File: localstack/services/acm/provider.py [hunk 1]
```diff
@@ -55,6 +56,32 @@ def describe(describe_orig, self):
...
 
 
+@singleton_factory
+def patch_cert_bundle_pickling():
+    """Apply patches to pickle CertBundle base models. It prevents the "cannot pickle 'builtins.Certificate'" errors"""
+    from cryptography.hazmat.backends import default_backend
+    from cryptography.x509 import load_pem_x509_certificate
+    from moto.acm.models import CertBundle
+
+    def cert_set_state(self, state, *args, **kwargs):
```
Comment (alexrashed) [inline localstack/services/acm/provider.py:66]: I think you could also use the decorator in `localstack.utils.patch` for this:
```
@patch(CertBundle.__setstate__)
def cert_set_state(self, state, *args, **kwargs):
  ...
```

**Review by alexrashed — APPROVED** [review]
LGTM! I just added a small comment concerning the usage of a utility function instead of "manually" patching the functions. 🚀 
FYI: I restarted the pipeline (seemed like a flake when building the Docker image).

**Review by whummer — APPROVED** [review]
Nice catch on moving this here, to support Community Cloud Pods 👍

File: localstack/services/acm/provider.py (hunk 1 above)
Comment (giograno) [reply localstack/services/acm/provider.py:66]: Could our decorator be used this way here? `acm.moto.CertBundle` does not have `__setstate__`

Comment (whummer) [reply localstack/services/acm/provider.py:66]: Great catch - I upvoted @alexrashed 's comment a bit too quickly, :) but there was a reason why we didn't use `@patch` previously! (as this code is copied/migrated from -ext)

Comment (giograno) [reply localstack/services/acm/provider.py:66]: 👍 thanks for the clarification @whummer

**Pull request merged**